workers/unity/Assets/Gdk/Android/Editor/AndroidSpatialRunner.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Improbable.Gdk.Android.Editor
+{
+    internal class AndroidSpatialRunner
+    {
+        private static readonly string ProjectDirectory = Path.Combine(Application.dataPath, "..", "..", "..");
+        private const string ProcessName = "spatial";
+        private const string ProcessParams = "local launch";
+        private const string Config = "android_launch.json";
+
+        [MenuItem("Improbable/Run Spatial for Android")]
+        public static void RunAndroidSpatial()
+        {
+            Process.Start(GetProcessStartInfo());
+        }
+
+        private static IPAddress GetLocalIpAddress()
+        {
+            var host = Dns.GetHostEntry(Dns.GetHostName());
+            var ipAddress = host.AddressList.FirstOrDefault(ip => ip.AddressFamily == AddressFamily.InterNetwork);
+            if (ipAddress == null)
+            {
+                throw new NullReferenceException(
+                    "Could not find local IP Address. Make sure you are connected to the Internet.");
+            }
+
+            Debug.Log($"Runtime IP: {ipAddress}");
```

Comment:
Is this debug print needed?

Reply:
It's for convenience because you have to type the IP manually on the device and this lets you quickly see it in the console (i.e. not have to run ```ipconfig``` in terminal).

Reply:
If this is something the user has to type in on the device maybe state that in the log message?